core/services/keeper/delegate.go

```diff
@@ -104,6 +104,7 @@ func (d *Delegate) ServicesForSpec(spec job.Job) (services []job.Service, err er
 		chain.TxManager().GetGasEstimator(),
 		svcLogger,
 		chain.Config(),
+		chain.Config().KeeperCheckUpkeepGasPriceFeatureEnabled(),
```

Comment:
This seems unnecessary - it's already on the config, why not simply read it from there?

Reply:
ah you're right